alpha/scripts/utils/moveFileSyncToDifferentStorage.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+define("BASE_DIR", dirname(__FILE__));
+require_once(BASE_DIR.'/../../../alpha/scripts/bootstrap.php');
+
+if (count($argv) != 4)
+{
+	KalturaLog::debug("USAGE: <partnerId> <storageId> <realrun-dryrun> ");
+	exit(0);
+}
+
+$partnerId = $argv[1];
+$storageId = $argv[2];
+if (!$storageId)
+{
+	KalturaLog::debug(" No Stroge Id");
+	exit(0);
+}
+
+$realRun = isset($argv[3]) && $argv[3] == 'realrun';
+if ($realRun)
+{
+	KalturaLog::debug("*************** In Realrun mode ***************");
+}
+else
+{
+	KalturaLog::debug("*************** In Dry Run mode ***************");
+}
+
+main($partnerId, $storageId, $realRun);
+
+/**
+ * @param $partnerId
+ * @param $storageId
+ * @param $realRun
+ * @throws PropelException
+ */
+function main($partnerId, $storageId,$realRun)
+{
+	KalturaLog::debug("Running for PartnerId [$partnerId] and storageId [$storageId]");
+	$partner = PartnerPeer::retrieveByPK($partnerId);
+	if (!$partner)
+	{
+		KalturaLog::debug("Partner [$partnerId] does not exists ");
+		exit(0);
+	}
+
+	$criteria = new Criteria(FileSyncPeer::DATABASE_NAME);
+	$criteria->add(FileSyncPeer::PARTNER_ID, $partnerId, Criteria::EQUAL);
+	$criteria->add(FileSyncPeer::STATUS, FileSync::FILE_SYNC_STATUS_READY, Criteria::EQUAL);
+	$criteria->add(FileSyncPeer::OBJECT_TYPE, FileSyncObjectType::ASSET);
+	$criteria->add(FileSyncPeer::OBJECT_SUB_TYPE, flavorAsset::FILE_SYNC_FLAVOR_ASSET_SUB_TYPE_ASSET);
```

Comment:
Add property to the args of file sync updatedat< 
As we do not want to touch files that have been updated in the last 24 hours (or more).
This should be an argument to the script <file minimal age in seconds>